File: 251109/약수가 세 개인 수/numbers-with-three-divisors.js
const fs = require('fs');
const input = fs.readFileSync(0).toString().trim().split(' ');
const [start, end] = input.map(Number);

let count = null;
let result = 0;
for (let i = start; i <= end; i += 1) {
    count = 0;
    for (let j = 1; j <= i; j += 1) {
        if (i % j === 0) count += 1;
    }
    if (count === 3) result += 1;
}

console.log(result);

File: 260314/약수가 세 개인 수/numbers-with-three-divisors.py
import sys

input = sys.stdin.readline

# 선언부
def solve(start: int, end: int):
    count = 0
    for i in range(start, end + 1):
        temp_count = 0
        for j in range(2, i):
            if i % j == 0:
                temp_count += 1
        if temp_count == 1:
            count += 1
    return count
    
# 구현부
start, end = map(int, input().split())
print(solve(start, end))